server/belga/macros/brief_internal_routing.py

```diff
@@ -135,6 +135,12 @@ def brief_internal_routing(item: dict, **kwargs):
     update_schedule_settings(item, PUBLISH_SCHEDULE, item[PUBLISH_SCHEDULE])
     item[PUBLISH_SCHEDULE] = item[PUBLISH_SCHEDULE].replace(tzinfo=None)
 
+    # Do not auto publish between 4 to 7 am
+    publish_time = utc_to_local(item[SCHEDULE_SETTINGS]['time_zone'], published_at).time()
```

Comment:
we should rather use `app.config["DEFAULT_TIMEZONE"]` as that item time zone could be different